src/kernel-install/50-depmod.install

```diff
@@ -2,7 +2,14 @@
 # -*- mode: shell-script; indent-tabs-mode: nil; sh-basic-offset: 4; -*-
 # ex: ts=8 sw=4 sts=4 et filetype=sh
 
-[[ $1 == "add" ]] || exit 0
 [[ $2 ]] || exit 1
 
-exec depmod -a "$2"
+if [[ $1 == "add" ]]; then
+    exec depmod -a "$2"
+fi
+
+if [[ $1 == "remove" ]]; then
+    exec rm -f /lib/modules/$2/modules.{alias{,.bin},builtin.bin,dep{,.bin},devname,softdep,symbols{,.bin}}
+fi
+
+exit 1
```

Comment:
Hm, so far unknown "verbs" were simply ignored and 0 was returned. I think that's the right thing to do, because it allows possible addition of new verbs in the future. We should also add this to man page.